charts/kube-state-metrics/values.yaml

```diff
@@ -304,8 +304,15 @@ containerSecurityContext:
 nodeSelector: {}
 
 ## Affinity settings for pod assignment
+## Can be defined as either a dict or string.  String is useful for `tpl` templating.
 ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
-affinity: {}
+affinity: ""
+  # podAntiAffinity:
+  #   requiredDuringSchedulingIgnoredDuringExecution:
+  #     - labelSelector:
+  #         matchLabels:
+  #           {{- include "kube-state-metrics.selectorLabels" . | nindent 12 }}
+  #       topologyKey: kubernetes.io/hostname
```

Comment:
The shouldn't work, we either need the multi-line strings notation or remove the templating.

Reply:
updated defaults.  what do you think?